bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java

```diff
@@ -1835,6 +1914,288 @@ public int runCmd(CommandLine cmdLine) throws Exception {
     }
 
     /**
+     * Command to trigger AuditTask by resetting lostBookieRecoveryDelay to its current value
+     */
+    class TriggerAuditCmd extends MyCommand {
+        Options opts = new Options();
+
+        TriggerAuditCmd() {
+            super(CMD_TRIGGERAUDIT);
+        }
+
+        @Override
+        String getDescription() {
+            return "Force trigger the Audit by resetting the lostBookieRecoveryDelay";
+        }
+
+        @Override
+        String getUsage() {
+            return CMD_TRIGGERAUDIT;
+        }
+
+        @Override
+        Options getOptions() {
+            return opts;
+        }
+
+        @Override
+        public int runCmd(CommandLine cmdLine) throws Exception {
+            ZooKeeper zk = null;
+            LedgerUnderreplicationManager underreplicationManager = null;
```

Comment:
same concerns above

It would be good to put this trigger logic in BookKeeperAdmin, for two seasons:

it keeps the shell tool simpler
it is easier to test and verify the logic.